site/src/content/docs/utilities/text.mdx

```diff
@@ -92,6 +92,12 @@ Change the line height with `.lh-*` utilities.
 <p class="lh-base">This is a long paragraph written to show how the line-height of an element is affected by our utilities. Classes are applied to the element itself or sometimes the parent element. These classes can be customized as needed with our utility API.</p>
 <p class="lh-lg">This is a long paragraph written to show how the line-height of an element is affected by our utilities. Classes are applied to the element itself or sometimes the parent element. These classes can be customized as needed with our utility API.</p>`} />
 
+## Balanced Text
+
+Use the `.text-balanced` class to balance text.
+
+<Example code={`<p class="text-balanced">his is a long paragraph written to show how the text-balanced of an element is affected by our utilities. Classes are applied to the element itself or sometimes the parent element. These classes can be customized as needed with our utility API.</p>`} />
```

Comment:
Looks like a typo here, at the start of the sentence - his should be This

Reply:
Fixed. Thanks.